identifier/src/containers/Login/Login.jsx

```diff
@@ -83,6 +84,17 @@ function Login(props) {
       history.replace(`/chooseaccount${history.location.search}${history.location.hash}`);
       return;
     }
+
+    if (query && query.login_hint){  // for login hint user name validation
```

Comment:
Space is missing after ), before { - please pay attention to coding style. Also further down below plenty of the same issues also spaces missing after if.

Reply:
Updated and made consistent with the above code.

Reply:
OK, i still see at least 4 missing spaces in the few lines below. What about those?

Reply:
I have added those as a seperation between the logics. Removed them as mentioned.